.github/workflows/windows-tests.yml

```diff
@@ -0,0 +1,28 @@
+name: Windows-specific tests
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+      - eculver/ci-nightly-windows-tests # This is a temporary hack to get the workflow to run on my PR branch
+jobs:
+  python-tests:
+    name: Python tests (Windows)
+    uses: ./.github/workflows/_python-tests.yml
+    with:
+      property_testing_preset: 'normal'
+      platform: '16core-64gb-windows-latest'
+      env-file: 'compose-env.windows'
```

Comment:
nit: this input is a little obtuse, might be nice if `./.github/workflows/_python-tests.yml` automatically inferred it

Reply:
yeah, we can infer it. I'll update.